cirq-google/cirq_google/devices/xmon_device.py

```diff
@@ -45,6 +45,23 @@ def __init__(
         self._exp_w_duration = cirq.Duration(exp_w_duration)
         self._exp_z_duration = cirq.Duration(exp_11_duration)
         self.qubits = frozenset(qubits)
+        self._metadata = cirq.GridDeviceMetadata(
```

Comment:
As with `SerializableDevice`, I would expect this to be `DeviceMetadata` or for `XmonDevice` to be a subclass of `GridDevice`.